tests/specs/fmt/unstable_yaml/__test__.jsonc

```diff
@@ -0,0 +1,25 @@
+{
+  "tempDir": true,
+  "tests": {
+    "nothing": {
+      "args": "fmt",
+      "output": "Checked 1 file\n"
```

Comment:
It's still loading the yaml file. I didn't think it was worth the effort to make that conditional as we'll be enabling this by default soon.